src/System.ComponentModel.TypeConverter/tests/SampleClasses.cs

```diff
@@ -82,13 +82,7 @@ public MyTypeListConverter(Type[] types)
         }
     }
 
-#if FUNCTIONAL_TESTS
-    [TypeConverter("System.ComponentModel.Tests.BaseClassConverter, System.ComponentModel.TypeConverter.Tests, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null")]
```

Comment:
Yes this is likely a binding redirect issue on desktop. You either need to make the version match exactly or add a binding redirect to lift up the version to the one loaded.